pkg/interactive/bubbletea/machinepoolbubble/survey_delegate.go

```
@@ -0,0 +1,145 @@
+	} else {
+		line = style.Render(line)
+	}
+
+	fmt.Fprint(w, line)
+}
+
+func itemTitle(item list.Item) string {
+	switch i := item.(type) {
+	case list.DefaultItem:
+		return i.Title()
+	case optionItem:
```

golangci-lint (2.12.2)

[error] SA4020: unreachable case clause: github.com/charmbracelet/bubbles/list.DefaultItem will always match before github.com/openshift/rosa/pkg/interactive/bubbletea/machinepoolbubble.optionItem

(staticcheck)